init.go

```diff
@@ -12,14 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// +build !windows
+// +build !windows,!journald
```

Comment:
The default behavior on Linux should not change and should not require a new build tag

Reply:
> The default behavior on Linux should not change and should not require a new build tag

Do you have any better idea for getting rid this useless code ?

On *LINUX* this code does pretty much NOTHING (throwing errors) when running as direct child of init
(that alone is just a ridiculously unreliable way to check for being a daemon) - that only works on Poetterix.

Reply:
Adding `no_journald` tag might be fine (if it has significant benefit on footprint)